app/domain/logs.rb

```diff
@@ -154,6 +154,11 @@ module AuthnK8s
         code: "CONJ00015D"
       )
 
+      VerifyCopySSLToPod = ::Util::TrackableLogMessageClass.new(
+        msg:  "Verifying SSL certificate is present in {0-container-name}:{1-cert-file-path}",
```

Comment:
On the client side, Micah used the word `available` instead of `present`. Maybe we should align?